packages/manager/modules/core/src/translate/translate.service.js

```diff
@@ -121,6 +121,15 @@ export default class TranslateServiceProvider {
     return this.LANGUAGES.defaultLoc;
   }
 
+  /**
+   * BCP 47 (also known as IETF language tag) is an international standard to identify human languages
+   * @param {string} language The language to convert, in the OVHcloud format (i.e.: 'fr_FR')
+   * @returns {string} The languag converted to BCP 47 (i.e.: 'fr-FR')
+   */
+  static convertFromOVHToBCP47(language) {
```

Comment:
👍 

maybe rename to something more explicit like ```ovhLanguageToBCP47``` ?

Reply:
You already are in the `Translate` service, hopefully methods in there will only service to language handling